docs/usage/AccessingBlueprints.md

```diff
@@ -0,0 +1,74 @@
+# Remote Access
+
+Blueprints are referenced in installations or installation templates via the component descriptors access.
+
+Basically blueprints are a filesystem, therefore, any blob store could be supported.
```

Comment:
I would remove this sentence - I was confused by it.

Reply:
I rephrased it to be more clear. WDYT?

Reply:
For me the part with the filesystem was confusing at the beginning. What about adding (not sure if this is technically ok): Basically, blueprints are a filesystem, i.e. a folder containing a file named "blueprint.yaml" with the blueprint manifest and additional files and subfolders containing whatever needed, like configuration data etc.